content/versions.yml

```diff
@@ -1,52 +1,52 @@
 - title: '16.12.0'
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#16120-november-14-2019
-- title: '16.11'
+- title: '16.11.0'
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#16110-october-22-2019
 - title: '16.10.2'
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#16102-october-3-2019
 - title: '16.10.1'
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#16101-september-28-2019
-- title: '16.10'
+- title: '16.10.0'
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#16100-september-27-2019
-- title: '16.9'
+- title: '16.9.0'
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1690-august-8-2019
-- title: '16.8'
+- title: '16.8.0'
   path: /version/16.8
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1680-february-6-2019
   url: https://5d4b5feba32acd0008d0df98--reactjs.netlify.com/
-- title: '16.7'
+- title: '16.7.0'
   path: /version/16.7
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1670-december-19-2018
   url: https://5c54aa429e16c80007af3cd2--reactjs.netlify.com/
-- title: '16.6'
+- title: '16.6.0'
   path: /version/16.6
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1660-october-23-2018
   url: https://5c11762d4be4d10008916ab1--reactjs.netlify.com/
-- title: '16.5'
+- title: '16.5.0'
   path: /version/16.5
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1650-september-5-2018
   url: https://5bcf5863c6aed64970d6de5b--reactjs.netlify.com/
-- title: '16.4'
+- title: '16.4.0'
   path: /version/16.4
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1640-may-23-2018
   url: https://5b90c17ac9659241e7f4c938--reactjs.netlify.com
-- title: '16.3'
+- title: '16.3.0'
   path: /version/16.3
   url: https://5b05c94e0733d530fd1fafe0--reactjs.netlify.com
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1632-april-16-2018
-- title: '16.2'
+- title: '16.2.0'
   path: /version/16.2
   url: https://5abc31d8be40f1556f06c4be--reactjs.netlify.com
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1620-november-28-2017
-- title: '16.1'
+- title: '16.1.0'
   path: /version/16.1
   url: https://5a1dbcf14c4b93299e65b9a9--reactjs.netlify.com
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1611-november-13-2017
-- title: '16.0'
+- title: '16.0.0'
   path: /version/16.0
   url: https://5a046bf5a6188f4b8fa4938a--reactjs.netlify.com
   changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1600-september-26-2017
-- title: '15.6'
+- title: '15.6.0'
```

Comment:
I think these links are OK to change in principle, except for this one that is actually for 15.6.3